docs/quantum-mechanic/command-reference.md

```diff
@@ -1,76 +1,352 @@
+2. Invite the Atomist Bot to the Team Channel.
+
+The Atomist Bot should present a generic message about creating a DevOps environment whenever it joins a new channel that is associated to a Team.
+
+**\*Note:** If you want to use a private Slack channel, the Atomist bot cannot be invited automatically. You will need to manually invite the bot the to channel using `\invite @Atomist` once you have linked the Team to the Slack channel.
+
+---
+#### `sub list projects`
+This command is used to provide users a way to view the details around what Projects are linked to a particular team. These details include the Project names and associated Bitbucket projects, along with options to drill down into each Project's associated Package details.
+
+---
+#### `sub list team members`
+This will present a list of the Team owners and users. This is useful to know who can perform actions that require ownership permission.
+
+---
+#### `sub onboard me` 
+This is the first command any user should run. Without on-boarding yourself you will not be able to run any subatomic commands. You will need to input your `first name`, `last name`, `email address` and `domain username`. Once submitted you will be able to execute subatomic commands granted you have the permission to execute them. For a detailed walk through click [here](../user-guide/onboarding.md).
+
+---
+#### `sub patch s2i image`
+This looks up the latest versions of s2i ImageStream's available, tags a selected one to your Team DevOps environment, and changes a selected Application's BuildConfig to use updated ImageStream.
+
+---
+#### `sub project request jenkins job`
+This requests a build folder for a specific project be created. This can either be useful if the build folder is deleted, or a user wants to create builds for a project without creating the OpenShift environments. The use case for the latter being when a Project only contains libraries and no deployable applications. This performs the following operations:
+
+1. Create the build folder for the Project. The build folder uses the Project's name to name the build folder.
+2. Create the environment name credentials within the build folder/job domain. These credentials store the OpenShift namespaces for each environment in your pipeline. E.g. "DEV_PROJECT" - Dev OpenShift Project, "TZA_UAT_PROJECT" - UAT OpenShift project for the TZA deployment pipeline.
+3. Create the Bitbucket authorization credential. This is used by Jenkins to clone sources from the configured Bitbucket instance.
+
+---
+#### `sub remove team member` 
+This command is used to remove a member from a Team. When removing a member from a team, the following actions are performed:
+
+1. Remove the Subatomic association between the user and the Team.
+2. Remove the user's permissions from all OpenShift namespaces associated to the Team.
+3. Remove the user's permissions from the Bitbucket project's associated to the Team.
+4. Kick the user from the Team's Slack channel.
+
```

Comment:
Perhaps a note about removing team owners?

**Note**: Team Owners can only be removed by a Team Owner (including themself) and may not remove themself if there are no other Team Owners in the team.